lib/view/home_screen.dart

```diff
@@ -38,7 +38,9 @@ class _HomePageState extends State<HomePage> {
       initialIndex: 0,
       length: 3,
       child: Scaffold(
-        drawer: DrawerWidget(),
+        drawer: SizedBox(
+            width: MediaQuery.of(context).size.width * Constants.drawerWidthPercentage,
```

Comment:
Apologies for answering with a question but did we discuss reducing the width of the drawer on the planning meeting? 

I can't recall exactly, but I perceeded with an impression that we did and intended to make the drawer narrower (which didn't happen due to too high `drawerWidthPercentage` const)

I can reduce `drawerWidthPercentage` to 0.6 if we want to have a narrower drawer or reverse it to the version from before my changes if we want to keep the current size.

p.s. I'd like to suggest here taking down requirements while planning due to the apparent troubles with memory of mine :)

Reply:
The problem was on nokia 1 with the height of the drawer, but you fixed it by removing an image and resizing the logo. The width was never a problem, since it is adjusted automatically to some fixed percentage of the screen width. I think we can leave it with default percentage.

I agree about taking down the precise requirements at the time of planning. :)